org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/IdeProjectDescriptor.xtend

```diff
@@ -227,6 +227,7 @@ class IdeProjectDescriptor extends ProjectDescriptor {
 													<exclude>*:�config.ideProject.name�-com.google.guava*</exclude>
 													<exclude>*:�config.ideProject.name�-asm*</exclude>
 													<exclude>*:�config.ideProject.name�-log4j*</exclude>
+													<exclude>*:�config.ideProject.name�-reload4j*</exclude>
```

Comment:
viel hilft viel ?

Reply:
That how it appears to be, yes...

Sadness. But since these are only excludes, it's probably fine.